_includes/homepage/homepage-usedby-band.html

```diff
@@ -69,6 +69,10 @@ <h2 class="text-centered blue">Strimzi is used by</h2>
       <div class="item__usedby">
         <img src="{{site.baseurl}}/assets/images/alauda.logo.png" />
       </div>
+      <div class="item__usedby">
+        <img src="{{site.baseurl}}/assets/images/randoli-logo.svg" />
```

Comment:
I think we need to fix the extension. (Or add the SVG image instead of the PNG)

```suggestion
        <img src="{{site.baseurl}}/assets/images/randoli-logo.png" />
```

Reply:
Ok, I guess you just fixed it :-)